docs/New Game.md

```diff
@@ -31,14 +31,82 @@ game to have proper pickups:
 2. Update `standard_pickup_configuration` and `ammo_pickup_configuration` in your game's starter preset.
 
 
+## Exporting games
 
-### Exporting games
+Up to now, everything is just theoretical values inside Randovania. The next step is making these modifications be
+applied to the game. 
 
-In order to export a game, a patcher is required. Typically, the patcher will be included in Randovania for seamless exports. This is a requirement for non-experimental games. Experimental games, however, may prefer an alternative implementation: refer to Corruption as an example of exporting data to be passed to a standalone patcher.
+In most case, this involves readings the files in
+a user-provided copy of the game and creating either a new ROM/ISO file or a mod file.
 
-**TODO**: details
+This process is split into two parts:
+
+### Patch Data Factory
+
+Each game defines a class that inherits from `PatchDataFactory`. Given a LayoutDescription, a CosmeticPatches and 
+PlayersConfiguration, it creates a JSON Object that will be passed as an argument to the Game Exporter.
```

Comment:
Would make `LayoutDescription`, `CosmeticPatches` and `PlayersConfiguration` in code blocks. Bonus points for linking to glossary.

Also technically it creates a dictionary, not a json object. Important distinction, as it does mean you can reformat it in other ways, should your patcher for some reason not deal with json nicely.

Reply:
No, it needs to be json serializable. 
A dict is considerably more powerful than just a json object.

Reply:
Return value from PatchDataFactory. But also whatever you pass to the patcher (which I called the "patcher data") as this is what often gets saved to a patcher.json or whatever.

Reply:
See how in later in the documentation, we talk about receiving a JSON as argument, as well as having a schema?

While you _could_ do that, that's kinda just real ugly and bad. Borderline acceptable if someone is integrating an existing patcher, but I'd also just recommend to refactor it to accept a better format. Worse case, just keep that _inside_ their patcher package.

Reply:
Cool. Was mainly just wondering if someone would try to write a patcher in some language whose json support isn't great for whatever reason, but now that I think about it the only languages where that *is* the case, wouldn't be accepted in RDV anyway.